raiden/network/rpc/client.py

```diff
@@ -320,8 +321,8 @@ def deploy_solidity_contract(
             self,  # pylint: disable=too-many-locals
             contract_name,
             all_contracts,
-            libraries,
-            constructor_parameters,
+            libraries=dict(),
```

Comment:
This will be the same object all the time, I think we should make this `None` and create a new list if it's `None` in the body.